adaboost/cuda/core/operations.hpp

```diff
@@ -4,6 +4,10 @@
 #include<adaboost/cuda/core/cuda_data_structures.hpp>
 using namespace adaboost::cuda::core;
 
+
+template <typename data_type_vec, typename data_type_ret>
+using func_t = data_type_ret(*)(data_type_vec);
+
```

Comment:
Put this inside namespace `adaboost::cuda::core`.

Reply:
Done